hbase-server/src/test/java/org/apache/hadoop/hbase/client/TestAdmin2.java

```diff
@@ -864,4 +865,42 @@ public void testSlowLogResponses() throws Exception {
     Assert.assertEquals(onlineLogRecords.size(), 0);
   }
 
+  @Test
+  public void testGetRegionServers() throws Exception {
+    // get all live server names
+    List<ServerName> serverNames = new ArrayList<>(ADMIN.getRegionServers(true));
+    Assert.assertEquals(3, serverNames.size());
+
+    List<ServerName> serversToDecom = new ArrayList<>();
+    ServerName serverToDecommission = serverNames.get(0);
+
+    serversToDecom.add(serverToDecommission);
+    ADMIN.decommissionRegionServers(serversToDecom, false);
+    waitForServerDecom(serverToDecommission, true);
+
+    Assert.assertEquals(2, ADMIN.getRegionServers(true).size());
+
```

Comment:
Add `Assert.assertEquals(3, ADMIN.getRegionServers(false).size());` for paranoid reviewers (cough cough me).

Reply:
Sure, definitely